designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessActivitySpec.scala

```diff
@@ -0,0 +1,201 @@
+package pl.touk.nussknacker.ui.api
+
+import io.restassured.RestAssured.`given`
+import io.restassured.builder.MultiPartSpecBuilder
+import io.restassured.http.ContentType
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import io.restassured.response.ValidatableResponse
+import org.hamcrest.core.IsEqual
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.engine.build.ScenarioBuilder
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.test.{NuRestAssureExtensions, NuRestAssureMatchers, RestAssuredVerboseLogging}
+import pl.touk.nussknacker.ui.api.helpers.TestCategories.Category1
+import pl.touk.nussknacker.ui.api.helpers.{
+  NuItTest,
+  NuScenarioConfigurationHelper,
+  TestProcessingTypes,
+  WithMockableDeploymentManager
+}
+
+import java.nio.charset.StandardCharsets
+import java.util.UUID
+
+class ProcessActivitySpec
```

Comment:
could you please take a look at `NodesApiSpec` and could we:
1. have a section with tests for each endpoint? 
2. have a test for each endpoint that checks if the user is authorized? 
3. have a test that checks invalid input (eg. non-existent process) for these endpoints it makes sense?

Reply:
1. 2. 3. -> Done.

According to failing slow test I will drop te pull request and I will submit new one.